.drone.star

```diff
@@ -567,6 +587,9 @@ def e2eTests(ctx):
         if "app-provider" in suite and not "full-ci" in ctx.build.title.lower() and ctx.build.event != "cron":
             continue
 
+        if "ocm" in suite and not "full-ci" in ctx.build.title.lower() and ctx.build.event != "cron":
+            continue
+
```

Comment:
ocm pipeline is still running

Reply:
ocm has been triggering because PR has full-ci in title